sale_customer_rank/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2021 ForgeFlow S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Sale Customer Rank",
+    "summary": "Update Customer Rank when creating a Sale Order",
+    "version": "14.0.1.0.0",
+    "category": "Sales",
+    "website": "https://github.com/OCA/partner-contact",
+    "author": "ForgeFlow, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "installable": True,
+    "depends": ["sale"],
```

Comment:
change for sale_management so it is installed

Reply:
Sure, done! :+1: